src/components/selectEvent/HexathonModal.tsx

```diff
@@ -38,48 +44,65 @@ const HexathonModal: React.FC = () => {
           emailHeaderImage: cdnUrl + emailHeaderImageID,
           coverImage: cdnUrl + coverImageID
       }
+    ) : await axios.put(
+      apiUrl(Service.HEXATHONS, `/hexathons/${props.hexathon.id}`),
+      {
+          name: data.name,
+          shortCode: data.shortCode,
+          isActive: data.isActive,
+          startDate: `${data.startDate}T00:16:30.934Z`,
+          endDate: `${data.endDate}T00:16:30.934Z`,
+          emailHeaderImage: cdnUrl + emailHeaderImageID,
+      }
     );
 
     onClose();
   }
 
+  let button;
+  if (props.num === 0) {
+    button = <Button onClick={onOpen}>Create Hexathon</Button>;
+  } else {
+    button = <Button onClick={onOpen}><EditIcon /></Button>;
```

Comment:
Let's make this button have the same height as the event card so it looks better. Should be 148px